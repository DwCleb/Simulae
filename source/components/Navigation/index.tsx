export { default as BackButton } from './BackButton';
export { default as DrawerContent } from './Drawer';
export { default as MenuButton } from './MenuButton';
