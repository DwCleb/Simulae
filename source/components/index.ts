export { default as KeyboardAvoidingView } from './Structure/KeyboardAvoidingView';
export { default as Typography } from './UI/Typography';
export { default as Button } from './UI/Buttons/Button';
export { default as SectionButton } from './UI/Buttons/SectionButton';
export { default as ExpandedHeader } from './UI/ExpandedHeader';
export { default as Carousel } from './UI/Carousel';
export { default as CarouselQuestionCard } from './UI/Carousel/CarouselQuestionCardrd';
export { default as Stepper } from './UI/Stepper';
export { default as ModalScreen } from './UI/Modal';
export { default as AlertModal } from './UI/AlertModal';
export { default as Input } from './UI/Input';
export { BackButton, DrawerContent, MenuButton } from './Navigation';
