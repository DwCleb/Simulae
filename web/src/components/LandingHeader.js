import React, { Fragment } from 'react';

const LandingHeader = () => {
  return (
    <Fragment>

      <header>
        <nav className="navbar navbar-default navbar-fixed-top">
          <div className="container-fluid">
              <div className="navbar-header"><a className="navbar-brand navbar-link" href="https://www.instagram.com/exp__designer/" target="_blank">Dize Design</a>
                  <button className="navbar-toggle collapsed" data-toggle="collapse" data-target="#navcol-1"><span className="sr-only">Toggle navigation</span><span className="icon-bar"></span><span className="icon-bar"></span><span className="icon-bar"></span></button>
              </div>
              <div className="collapse navbar-collapse" id="navcol-1">
                  <ul className="nav navbar-nav navbar-right">
                      <li role="presentation"><a href="#" className="scroll-link" data-id="page-top">Back to Top</a></li>
                  </ul>
              </div>
          </div>
        </nav>
        <div className="container">
          <div className="row">
            <div className="col-md-12">
              <h1>sky climber's </h1>
              <p>Climb up with us. </p>
            </div>
            <div className="col-md-12">

              <section id="section05" className="demo arrow"><center>
              <h1></h1><a href="#" className="scroll-link" data-id="go"><span></span>Scroll Down</a></center></section>

            </div>
          </div>
        </div>
      </header>
      <section id="go">
        <div class="container">
            <div class="row">
                <div class="col-md-12">
                    <h1>or make youre way</h1>
                    <p>Be Creative</p>
                </div>
            </div>
        </div>
    </section>
    </Fragment>

  );
};

export default LandingHeader;
