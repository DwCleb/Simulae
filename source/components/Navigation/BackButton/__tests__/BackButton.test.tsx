import React from 'react';
import { create } from 'react-test-renderer';

import { BackButton } from '~/components';

import { ThemeProvider } from 'styled-components/native';
import darkTheme from '~/theme/ThemeContext/dark';

let wrapper = {};

const props = {};

const { theme } = darkTheme;

const action = jest.fn();

describe('<BackButton />', () => {
  beforeAll(() => {
    jest.useFakeTimers();
  });

  beforeEach(() => {
    wrapper = create(
      <ThemeProvider theme={theme}>
        <BackButton action={(): object => action} {...props} />
      </ThemeProvider>
    );
    return wrapper;
  });
  it('Should be render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
