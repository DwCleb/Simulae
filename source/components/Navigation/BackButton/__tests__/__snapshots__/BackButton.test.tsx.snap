// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BackButton /> Should be render correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "marginLeft": 40,
      "marginTop": 0,
      "opacity": 1,
      "paddingBottom": 0,
      "paddingLeft": 0,
      "paddingRight": 40,
      "paddingTop": 0,
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../src/assets/icons/back.png",
      }
    }
    style={
      Array [
        Object {
          "height": 28,
          "tintColor": "#FFFFFF",
          "transform": Array [
            Object {
              "scaleX": 1,
            },
          ],
          "width": 16,
        },
      ]
    }
  />
</View>
`;
